src/js/controller/controller.js

```diff
@@ -354,17 +369,16 @@ Object.assign(Controller.prototype, {
                     } else {
                         _play({ reason });
                     }
-                } else if (OS.mobile && !_getAdState()) {
+                } else if (OS.mobile && !adState) {
                     _this.pause({ reason: 'autostart' });
                     _model.set('playOnViewable', true);
                 }
-            }
-        }
 
-        function _pauseWhenNotViewable(viewable) {
-            if (!viewable) {
-                _this.pause({ reason: 'viewable' });
-                _model.set('playOnViewable', !viewable);
+                if (OS.mobile && adState) {
+                    // If during an ad on mobile, we should always be paused after the ad,
```

Comment:
just want to be clear here, regardless of any autopause setting we _always_ want to pause after an ad? is this for a specific case? it feels to me like in most cases we would want to go straight to the actual content after an ad

Reply:
@jnatalzia If it's not viewable and `model.get('playOnViewable') === true`, then yes, the content should pause after the ad.

Reply:
ahhhh missed the `playOnViewable` with the weird diff 👍